libcodechecker/store.py

```diff
@@ -323,6 +348,11 @@ def main(args):
     if isinstance(args.input, str):
         args.input = [args.input]
 
+    compile_commands = {}
+    if len(args.logfile) != 1:
+        LOG.warning("Only one log file can be processed right now!")
+        compile_commands = process_compile_db(args.logfile[0])
```

Comment:
I've cleaned this up. No new argument will be available so no check is needed. The reports directory is checked for a `compile_cmd.json` file.